tdp/core/variables/cluster_variables.py

```diff
@@ -27,7 +28,11 @@ def __iter__(self):
 
     @staticmethod
     def initialize_cluster_variables(
-        collections, tdp_vars, override_folder=None, repository_class=GitRepository
+        collections,
+        tdp_vars,
+        override_folder=None,
+        repository_class=GitRepository,
+        validate=False,
```

Comment:
Why disable validation ?

Reply:
To make it opt-in, there might be use case where we define the schema wrongly, and we shouldn't prevent initialization is a tdp-lib user **realy** wants to initialize

Reply:
Yes, and this is a default value, the CLI call it with `True` by default.